config.yml

```diff
@@ -0,0 +1,42 @@
+script_options:
+  TESTNET: True
```

Comment:
Do you need a similar message as the one that was removed?

```
# Switch between testnet and mainnet
# Setting this to False will use REAL funds, use at your own risk
# Define your API keys below in order for the toggle to work
```

Reply:
Good idea. Lemme fix that.